test/config-luatest/instance_config_schema_test.lua

```diff
@@ -1210,3 +1198,54 @@ g.test_feedback_disabled = function()
                 'without feedback reports sending support'
     t.assert_equals(err, exp)
 end
+
+g.test_security_enterprise = function()
+    t.tarantool.skip_if_not_enterprise()
+
```

Comment:
Nit: I would add one success scenario for CE (just `security.auth_type = 'chap-sha1'`).

Reply:
Done.